docs/tutorials/cox_model/cox_model.ipynb

```diff
@@ -0,0 +1,893 @@
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "cph.summary.reset_index()[[\"covariate\", \"coef\", \"se(coef)\"]].merge(\n",
+    "    model_glum.coef_table(X=df_split, robust=False)[[\"coef\", \"se\"]],\n",
+    "    left_on=\"covariate\",\n",
+    "    right_index=True,\n",
+    ").rename(\n",
+    "    columns={\n",
+    "        \"coef_x\": \"coef_coxph\",\n",
+    "        \"coef_y\": \"coef_poisson\",\n",
+    "        \"se(coef)\": \"se_coxph\",\n",
+    "        \"se\": \"se_poisson\",\n",
+    "    }\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3. Speed Considerations<a class=\"anchor\"></a>\n",
+    "\n",
+    "The Poisson model estimates each time fixed effect parameter and, therefore, many more parameters than the Cox model. In our example, there are 61 coefficients in the Poisson model as opposed to 12 in the Cox model:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "(12, 61)"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "len(cph.summary), len(model_glum.coef_)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "One might, therefore, wonder if the Poisson approach is competitive in terms of estimation speed. For the dataset here, the Poisson approach, including the data transformation by `survival_split`, turns out to be faster than the Cox model. This speedup is likely aided by tabmat's optimizations for the high-dimensional `week` categorical."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "32.8 ms ± 3.79 ms per loop (mean ± std. dev. of 7 runs, 10 loops each)\n"
+     ]
+    }
+   ],
+   "source": [
+    "%timeit cox_model = CoxPHFitter().fit(df, duration_col='week', event_col='arrest', formula=\"fin + bs(age, df=4) + race + wexp + mar + paro + bs(prio,df=3)\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "18 ms ± 73.4 µs per loop (mean ± std. dev. of 7 runs, 100 loops each)\n"
+     ]
+    }
+   ],
+   "source": [
+    "%timeit model_glum = glum.GeneralizedLinearRegressor(family=\"poisson\",formula=\"arrest ~ fin + bs(age, df=4) + race + wexp + mar + paro + bs(prio,df=3) + C(week)\").fit(survival_split(df, 'week', 'arrest'))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Footnotes\n",
+    "<span id=\"fn1\"> 1</span>:\n",
+    "The Cox model assumes that at most one individual has an event at any time (\"no ties\").\n",
```

Comment:
Maybe mention here that there are various ways to deal with this? (e.g., exact - and expensive - calculations, efron approximation (reasonably accurate), breslow approximation (fastest). Also, in the case of many ties, an inherently discrete survival model might be the best option.